src/Components/App/App.js

```diff
@@ -16,11 +16,9 @@ function App() {
 
   const toggleTimerView = (hide) => {
     const homeContainer = document.getElementById('HomeContainer')
-    if (hide) {
-      homeContainer.classList.add('hidden')
-    } else {
-      homeContainer.classList.remove('hidden')
-    }
+    hide
```

Comment:
Sorry I'm being soo picky again, but I think convention for ternaries based on previous threads I was reading about ternaries v if/else is that the "?" should live on the first line - better for dev empathy as it's easier to recognize you're reading a ternary as soon as you see that "?" (E.g. here is the ES Lint guideline: https://eslint.org/docs/rules/multiline-ternary)

Reply:
Aha,interesting to see it this way in the es-lint docs! I formatted this the other way based on the airBnB style guide that I referenced in our previous discussion and other examples I saw in my research. But ultimately I think that we should do what makes most sense to us, and if there's a strong preference plz let me know!